### ISSUE TITLE
No helping text for icons

### ISSUE THREAD

**Issue by nikitaermishin** [llaske/sugarizer#173]
Here is an issue about tooltip text, which we should get near icons (I've pinned image below)

![2018-01-09 00-57-09](https://user-images.githubusercontent.com/26204565/34694834-985a63b0-f4d9-11e7-9799-bbb43a32721a.png)

I think that this bug is easy-fixing. It's very important, because it will help people interacting with Sugarizer.

**Comment by llaske** [comment]
Unable to reproduce. Could you tell us more about your test environment?

**Comment by nikitaermishin** [comment]
Hmm, I can't reproduce it again... I cathed it. It's first launch bug. I've reinstalled Sugarizer from chrome store and got it again.

**Comment by llaske** [comment]
Sorry unable to reproduce. Could you give use more information about your testing environment.

**Comment by nikitaermishin** [comment]
Yes, my OS is Ubuntu 16.04, browser - Chrome 65.0.3298.3 (Dev version bit). Try to reproduce it by reinstalling Sugarizer app in Chrome. After that write your nickname and set mouse on person's icon. Then you will see a popup with no text. Here is a GIF(in this GIF I've shown full problem (with reinstalling etc)):

![gif](https://user-images.githubusercontent.com/26204565/34890354-9cebbd0a-f7e2-11e7-9fa6-945ea6160a1c.gif)

**Comment by llaske** [comment]
I can't reproduce with the same process that you. I guess it's related to your default language (Russian ?).
Could you check if everything work when you force the language in Settings/My language ?

**Comment by llaske** [comment]
Finally reproduce it by tricking the Chrome language.
As I guessed the issue was related to language. It happens at first launch on Chrome OS when navigator language is not one of Sugarizer known language.
Fixed in https://github.com/llaske/sugarizer/commit/b07701d96fad92c280e0246abee535f994882e4e
